### DISCUSSION TITLE
license of punkt in nltk_data

### DISCUSSION THREAD

**Discussion by happyMindHaha** [community/community#19264] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi Nice to meet you.

Is it possible to use punkt in nltk_data for commercial use freely?
What is the license of punkt in nltk_data?
https://github.com/janstrunk/nltk_data/tree/gh-pages

Thank you.

**Comment by queenofcorgis** [comment, marked as answer]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

1. Check the README for instructions on how to operate the software, pointers to documentation or troubleshooting info.
2. Check the [SUPPORT file](https://help.github.com/articles/adding-support-resources-to-your-project/), if one exists for instructions on how to best contact the maintainers for support.
3. Check the [CONTRIBUTING guide](https://help.github.com/articles/setting-guidelines-for-repository-contributors/), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
